analyzers/tests/SonarAnalyzer.UnitTest/TestCases/ArrayPassedAsParams.cs

```diff
@@ -112,3 +112,22 @@ public void CallMethod()
     }
 }
 
+public class Repro6977
+{
+    // Reproducer for https://github.com/SonarSource/sonar-dotnet/issues/6977
```

Comment:
Pedantic: Move the comment above the class, like so:

```csharp
// Reproducer for https://blabla
public class ReproXXXX
{
```

It's slightly more concise and eliminates some white space